docs/UsingSpiffdemo/Basic_Tools.md

```diff
@@ -9,15 +9,23 @@ capabilities.
 Here are fundamental blocks of our system:
 
 ## Displaying Content
-The Spiffarena platform offers powerful features for displaying content  within your BPMN processes.
-Follow the steps below to effectively display content in your processes:
+The Spiffarena platform offers powerful features for displaying 
```

Comment:
Yeah basically I try to manually format it by giving line numbers to each line however when I change the size of windows then it breaks the lines as well. I think its because I use word to write the whole document first and then paste it here maybe that is the problem.

Reply:
we chatted offline on this. on same page.